README.md

```diff
@@ -1,9 +1,93 @@
 # Intro to Databases
 
+This repo is designed as a guideline for mentors so they can explain databases to begginner developers who are not familiar with the concept.
+
 ## What's a database, why use it in an application and when is it useful?
 
+### When and why to use a database?
+
+- Apps that we use in real life, save data, for example if an app has users then it'll save their login details.
+- This data has to be saved somewhere, a "data centre" where we save new data, fetch it or edit it.
+- This "data centre" is called a database.
+- You can simplify this concept by saying that a database is a chunk of memory dedicated for the app's data.
+
+### In which applications is it useful to have a database?
+
+- Ask the students this question to see if they have understood.
+- Then ask which applications would not need a database.
+
+### Example for a database
+
+- The example would be about users and posts (a small facebook).
+- You can start by explaining that the app has users where each user has a username and a password, whereas each post has a user reference and a post id.
+- Before writing this on the board, ask the students how they expect each data would be represented, as in which data does a user have and what does a post have.
+
 ## How to structure a database and how to separate it into tables?
 
+- After explaining what data you need, you can start talking about tables.
+- A table is a specific type of data, ask the students how would they separate the
+types of data in the example.
+- If the answer was that you should put the posts in the users table, explain that you shouldn't repeat the same information more than once and it would indicate the need for a new table, otherwise ask them to explain why they wouldn't put the posts next to a user's details.
+- Draw the tables on the board, explain what each column means and that each row is a data entry for a specific user/post.
+- Explain the need of an id: every table has a unique and not null (not empty) column which is the id of that entry, useful for referencing a specific row, when you have multiple tables.
+- The tables that you would use would look like this:
+
+Users table:
+
+|user_id|username|password|
+|-------|--------|--------|
+
+Posts table:
+
+|post_id|user_id|
+|-------|-------|
+
 ## What is a schema and how to make one?
 
+- Explain that a schema is the way we divide the database into tables.
+- so in the example the schema is the users table and the posts table.
+- Give another example of a database and ask the students to build the schema: a pets database, the database will have the pet owners with their name, address and phone number and their pets with the name and what type of animal is it.
+- Do these examples
```

Comment:
I feel like maybe talking about the different kinds of relationships completely abstracted away from code may be more useful, before talking about tables. 

I feel that getting an understanding one to many, many to many and one to one relationships before getting into how they actually look in terms of tables was quite useful when FAC10 were taught this stuff. 

Examples were things like:
one to one: mongamous romantic relationship
one to many: mother/father to child
many to many: person to their siblings, or writers on a tv show or something of the sort

We were offered the chance to come up with what might be an example of each of these kinds of relationships before being given these examples. 

Perhaps this could be added into this explanation?